docs/pre_release_features.md

```diff
@@ -46,30 +46,30 @@ If you'd like to review the changes for a nightly release, you can use the [comp
 
 ## Beta and Experimental Namespaces
 
-In addition to nightly releases, we're also introducing two new namespaces for Streamlit features: `st.beta` and `st.experimental`. These are basically prefixes we attach to our function names to make sure their status is clear to everyone.
+In addition to nightly releases, we're also introducing two new naming conventions for Streamlit features: `st.beta_` and `st.experimental_`. These distinctions are prefixes we attach to our function names to make sure their status is clear to everyone.
```

Comment:
Since these conventions have been available for a while, should we change the "introducing" text? E.g.

"In addition to nightly releases, we also have two conventions for {upcoming/newer/less stable} Streamlit features:"

Reply:
:+1: